### PR TITLE
Provided alternate solution for power by using a recursive approach #25

### PR DESCRIPTION
**Pull request by GaganMishra305**
Hey @shishiro26 @Illuminati9 @Kota-Karthik @akhilender-bongirwar , solved issue #25 . 
Provides an alternate approach to the power function by converting its implementation to recursive #25

### CODE REVIEW

**Comment by GaganMishra305** [comment]
Hey @shishiro26 @akhilender-bongirwar @Illuminati9 @Kota-Karthik the approach i went for solving this issue is pretty straightforward.
Instead of finding the power iteratively i went for a recursive approach which uses basic recursion for raising the base to a specified power.


Screenshot 2023-10-19 175204

This above written function by me is the direct implementation of recursive exponentiation algorithm and this image gives a pretty clear view of how it works-

Screenshot 2023-10-19 175629

**Review by akhilender-bongirwar — CHANGES_REQUESTED** [review]
@beforeIkillYou, remove all the commented code.

**Comment by GaganMishra305** [comment]
Hey @akhilender-bongirwar @Illuminati9 @PavanaSakethaRam , removed all the commented code from issue #25 and provided a solution using a recursive approach.

File: Students/LCI2023038/power.cpp [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    int exponent;
+    cin >> base;
+    cin >> exponent;
+    double result = Power(base, exponent);
+    cout << "Result: " << result << endl;
+    return 0;
+}
+
+// Provide an another/alternate approach to solve the above question 
+// (Do you want a hint?).
```
Comment (akhilender-bongirwar) [inline Students/LCI2023038/power.cpp]: @beforeIkillYou , remove these commented lines also.

**Review by akhilender-bongirwar — APPROVED** [review]

**Review by shishiro26 — CHANGES_REQUESTED** [review]
Please do follow the folder structure for the solutions. Make sure to create a folder for the type of problem you are approaching or solving. @beforeIkillYou . solve it asap

**Review by akhilender-bongirwar — CHANGES_REQUESTED** [review]
Follow folder structure you need to create a new folder "Alternate_Solution" inside LCI2023038 and then add power.cpp

**Comment by GaganMishra305** [comment]
Hello @akhilender-bongirwar @Illuminati9 @shishiro26 @Kota-Karthik , i completed all the required changes . Made the required folders and commented all the unnecessary code and and text.

**Review by shishiro26 — COMMENTED** [review]
@beforeIkillYou, you have created another 'power.cpp' file in the 'lci2023038' folder. Please remove the one in the 'lci2023038' folder.

File: Students/LCI2023038/power.cpp
Comment (shishiro26) [inline Students/LCI2023038/power.cpp:1]: Remove this file

**Comment by GaganMishra305** [comment]
Hello @akhilender-bongirwar @Illuminati9 @shishiro26 @Kota-Karthik , i completed all the required changes . Removed the 'power.cpp' file.

**Review by shishiro26 — APPROVED** [review]
LGTM 👏

**Review by akhilender-bongirwar — APPROVED** [review]

**Review by Illuminati9 — APPROVED** [review]
LGTM

**Review by PavanaSakethaRam — APPROVED** [review]

**Pull request merged**